.github/workflows/ci.yml

```diff
@@ -241,4 +241,4 @@ jobs:
       - run: npm run coverage
         env:
           CI: true
-      - run: bash <(curl -s https://codecov.io/bash)
+      - run: bash <(curl -s https://codecov.io/bash)
```

Comment:
could you please add the empty newline at the EOF

Reply:
Done